### PR TITLE
[Outlook] (Mac) Reference saveAsync workaround

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[db39c8d](https://github.com/OfficeDev/office-js-docs-pr/pull/1045/commits/db39c8ddc175a76b7bd13d9d326af1f5287efd6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905292224055355-1045%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=riEl8m496rB2KhdmHEAQvdOpcYaOQX9RmuqUe6OZJws%3D&st=2019-05-29T22%3A20%3A10Z&se=2019-06-29T22%3A25%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fa2382c](https://github.com/OfficeDev/office-js-docs-pr/pull/1045/commits/fa2382c8df3404702d52033b5fab8587933c6a1f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905301754151251-1045%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jiKY4OrQ%2F%2BCDvlvvJ1LsSJCayj5l2TSxb9VIgxVqqVE%3D&st=2019-05-30T17%3A50%3A20Z&se=2019-06-30T17%3A55%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6d4e8ef](https://github.com/OfficeDev/office-js-docs-pr/pull/1045/commits/6d4e8efc09fa3e9ab36991e087216aed9cf21567)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.4/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-mac-saveasync/docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905301829355837-1045%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aTw7mbshjiQMCFtCTeAgMooto008Y8j62vaNDWAxpec%3D&st=2019-05-30T18%3A25%3A50Z&se=2019-06-30T18%3A30%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md [hunk 1]
```diff
@@ -2404,7 +2404,7 @@ Since appointments have no draft state, if `saveAsync` is called on an appointme
 > [!NOTE]
 > The following clients have different behavior for `saveAsync` on appointments in compose mode:
 >
-> - Mac Outlook does not support `saveAsync` on a meeting in compose mode. Calling `saveAsync` on a meeting in Mac Outlook will return an error.
+> - Outlook for Mac does not support `saveAsync` on a meeting in compose mode. As such, calling `saveAsync` in that scenario returns an error. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.
```
Comment (davidchesnut) [inline docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md]: I recommend using this text from another PR you created. I like this wording better. I think it reads easier.

Outlook for Mac does not support saving a meeting. The `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.

Comment (davidchesnut) [reply docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md]: I recommend using this wording through the rest of the files in this PR.

**Pull request merged**